packages/fastify-renderer/src/node/types.ts

```diff
@@ -25,7 +23,9 @@ export interface FastifyRendererHook {
   name?: string
   tails?: () => string
   heads?: () => string
-  transform?: (app: ReactElement) => ReactElement
+  transform?: {
```

Comment:
Format change. Functions can't cross worker (thread) boundary, but an string to import can.